troposphere/static/js/actions/SSHKeyActions.js

```diff
@@ -0,0 +1,48 @@
+import SSHKeyConstants from "constants/SSHKeyConstants";
+import SSHKey from "models/SSHKey";
+import Utils from "./Utils";
+
+export default {
+    create: function({name, pub_key, atmo_user}) {
```

Comment:
Is this a destructure of the first argument passed in?  Or a destructure applied to `this.arguments` for the `Function` object? 

Curious, I don't think of noted this as a place we can use that.